add(platform): limited and early availability tags (#29)
aiven/aiven-docs: docs/platform/howto/use-google-private-service-connect.md

## Prerequisites

> :::important
> Google Private Service Connect is a
> [limited availability feature](/docs/platform/concepts/beta_services), which you can request from the sales team
> ([sales@aiven.io](mailto:sales@aiven.io)). During the limited availability stage, you can use
> the feature at no cost. If you want to continue using Google Private
> Service Connect after it reaches general availability, you\'ll be billed
> according to the latest applicable price.
> 
> Google Private Service Connect is supported for Aiven for Apache Kafka®
> and Aiven for ClickHouse® only.
> :::

Reviewer: ```suggestion
This feature is in limited availability. Contact the sales team(mailto:sales@aiven.io) to enable it.
```
